README.md

```diff
@@ -84,6 +84,5 @@ MyProject/MyProject.hxproj
 [--no-gitignore]         : Don't generate .gitignore
 [--no-readme]            : Don't generate README.md
 [--no-haxedevelop]       : Don't generate HaxeDevelop project files
-[--no-makefile]          : Don't generate a makefile
```

Comment:
The line I removed is actually a duplicate.  `--no-makefile` is first listed on line 83, then again on the line I removed.